src/pfe/file-watcher/server/test/functional-test/suites/project/tests/project-event.ts

```diff
@@ -35,7 +35,7 @@ export function projectEventTest(socket: SocketIO, projData: projectsController.
             "projectID": projData.projectID,
             "timestamp": Date.now(),
             "chunk": chunk,
```

Comment:
The `chunkNum` indicates number of chunks in `chunk_total`. It should be <= `chunk_total`

Since you only send 1 chunk in the update test, setting `chunk_total=1` is correct. But don't forget to also set `chunk=1`. Your original value is a random number.

Reply:
So how does is 20s timer not hit when I have `chunk` set to a random number and `chunk_total` is set to 1?

Reply:
Since when `chunk_total=1`, we would proceed the build directly. 
https://github.com/eclipse/codewind/blob/b0f16f0a76be2f543d501fc718ddeaa727d75d1d/src/pfe/file-watcher/server/src/controllers/projectEventsController.ts#L167-L168

But since this is the functional test, the `chunkNum` should be set to an appropriate value. 
Random number does not make any sense in the functional level.

Reply:
Alright, set `chunk` to 1 as well.